components/Button/ButtonPrimary.tsx

```diff
@@ -0,0 +1,20 @@
+import Button from "./buttonPrimary.style";
+import Link from "next/link";
+import React from 'react'
+
+interface Buttons {
+  text: string
+  href?: string
+}
+
+const ButtonPrimary = ({ text, href }:Buttons) => {
+  return (
```

Comment:
On the Button's props you can add the variant prop:
const Button = ({ text, href, variant }:ButtonTypes) => {

The variant can be primary, secondary, tertiary, etc...

On the styles file of the Button you can then have the diffent variants, something like that:

`export const Button = styled('a')(
  {
    padding: space.large
  },
    variant({
      variants: {
        primary: {
          background: color.primary
        },
        secondary: {
           background: color.secondary
        },
      },
    })
)`